test-files/article.html

```diff
@@ -76,9 +76,9 @@ <h3 class="code-title">Code</h3>
 			<h2 class="test-title" id="ar-004">Test ar-004: Article with <code>aria-labelledby</code></h2>
 
 			<div class="test-case">
-				<article aria-labelledby="ar-004-labelledby">
-					<h1 id="ar-004-labelledby">h1 referenced by aria-labelledby</h1>
-					<p>Aria-labelledby should be used as accessible name.</p>
+				<article aria-labelledby="ar-004-labelledby4">
+					<h1 id="ar-004-labelledby4">h1 referenced by aria-labelledby4</h1>
```

Comment:
some of these add the number directly to the end of the aria-labelledby (or whatever attribute), but the actual attribute that should win is `aria-labelledby` not `aria-labelledby4` or whatever number. The number should be added some other way such as a space like on line 64 or maybe in brackets. Either way they should be consistent for each one.